pwiz_tools/Skyline/SettingsUI/ViewLibraryDlg.cs

```diff
@@ -2713,44 +2722,58 @@ public Size RenderTip(Graphics g, Size sizeMax, bool draw)
 
                 using (RenderTools rt = new RenderTools())
                 {
-                    // Draw sequence
-                    sizeSeq = DrawTextParts(g, 0, 0, _seqPartsToDraw, rt);
-
-                    // Draw small mol info
-                    if (_smallMoleculePartsToDraw != null)
+                    if (!string.IsNullOrEmpty(_errorMessage))

```

Comment:
Is there a way to make this change with a small diff? Or maybe introduce sub-functions?